src/main/java/org/xstefank/webhook/WebHookEndpoint.java

```diff
@@ -44,14 +47,30 @@ public void processPullRequest(JsonNode payload) {
     }
 
     private static FormatConfig readConfig() {
-        String configFileName = System.getProperty(TEMPLATE_FORMAT_FILE);
-        if (configFileName == null) {
-            configFileName = System.getProperty(Utils.JBOSS_CONFIG_DIR) + "/format.yaml";
```

Comment:
so now you are loading format only from URL and and format.yaml in configuration. Please reverse the original property option